src/main/java/org/broadinstitute/dsde/consent/ontology/translate/service/Translate.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.dsde.consent.ontology.translate.service;
+
+import org.broadinstitute.dsde.consent.ontology.Utils;
+import org.broadinstitute.dsde.consent.ontology.translate.DTO.RecommendationDTO;
+import org.broadinstitute.dsde.consent.ontology.translate.model.TermItem;
+import org.broadinstitute.dsde.consent.ontology.translate.utils.LoadUtils;
+import org.broadinstitute.dsde.consent.ontology.translate.utils.SearchUtils;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class Translate {
+
+    String jsonFilePath;
+
+    public Translate(String jsonFilePath) {
+        this.jsonFilePath = jsonFilePath;
+    }
+
+    public HashMap<String, RecommendationDTO> paragraph(final String paragraph) throws Exception {
+        HashMap<String, RecommendationDTO> recommendations = new HashMap<>();
+
+        if (Utils.isNullOrEmpty(paragraph) || Utils.isNullOrEmpty(jsonFilePath)) {
+            return recommendations;
+        }
+
+        List<TermItem> terms = LoadUtils.termsFromLocalResourcesJSON(jsonFilePath);
```

Comment:
It looks like this list of terms is constant over the lifetime of the app. In that case, if it's not too large, I would load the list of terms once during `Translate`'s construction. The overhead of loading a json structure isn't great but there's no benefit in incurring this cost every time an API call is made. In the case where the list of terms is very large, you'd probably need a better way to perform an indexed search over terms.